src/Compilers/VisualBasic/Test/Emit/CodeGen/CodeGenWinMdEvents.vb

```diff
@@ -30,7 +30,7 @@ End Class
                 Diagnostic(ERRID.ERR_TypeRefResolutionError3, "E").WithArguments("System.Runtime.InteropServices.WindowsRuntime.EventRegistrationToken", "MissingReferences.winmdobj"))
 
             ' Throws *test* exception, but does not assert or throw produce exception.
-            Assert.Throws(Of EmitException)(Sub() CompileAndVerify(comp))
+            Assert.Throws(Of Exception)(Sub() CompileAndVerify(comp))
```

Comment:
Looks like asserts changed in this file no longer assert what the comment above them says - that the compilation doesn't throw a product exception.

Reply:
Good catch. Looked at this a bit and decided to just add back `EmitException`.